sdk/resourcemanager/azure-resourcemanager/src/test/java/com/azure/resourcemanager/AzureResourceManagerTests.java

```diff
@@ -238,6 +238,7 @@ public void run() {
      * @throws IOException
      * @throws ManagementException
      */
+    @DoNotRecord(skipInPlayback = true) // response contains token in hostpoolToken
```

Comment:
We can't redact this one?

Reply:
we may, but the response is 2+ MB, and I don't really want to figure out which part of the 2MB data has token (I can see one, but that might not be the only one).